CliFx/Services/CommandSchemaResolver.cs

```diff
@@ -31,7 +31,8 @@ private IReadOnlyList<CommandOptionSchema> GetCommandOptionSchemas(Type commandT
                     attribute.Name,
                     attribute.ShortName,
                     attribute.IsRequired,
-                    attribute.Description);
+                    attribute.Description,
+                    attribute.EnvironmentVariableName);
```

Comment:
Will also need a check to make sure there are no options with duplicate environment variable names?

Reply:
I cannot image an use case where binding multiple options to the same environment variable could be usefule. I guess it makes sense to forbid to use the same variable multiple times, although, given that it does not affect options parsing, we could leave it as it is for additional flexibility.

Reply:
Yeah I can imagine it could be used purposely, although it's more likely a user error. Let's not validate it for now.